PWG/EMCAL/EMCALbase/AliEMCALRecoUtils.cxx

```diff
@@ -505,13 +505,13 @@ Bool_t AliEMCALRecoUtils::CheckCellFiducialRegion(const AliEMCALGeometry* geom,
   Bool_t okrow = kFALSE;
   Bool_t okcol = kFALSE;
 
-  if (iSM < 0 || iphi < 0 || ieta < 0 )
-  {
+  if (iSM < 0 || iphi < 0 || ieta < 0 ) {
```

Comment:
I see this kind of change happening often in recent commits to common code, please this is "my" style. I prefer to leave spaces between lines for readability. Do not change if not really necessary, here or elsewhere. This will also easier to get to the point of the commit.